modules/claims_api/app/controllers/claims_api/metadata_controller.rb

```diff
@@ -6,14 +6,21 @@ class MetadataController < ::ApplicationController
     skip_after_action :set_csrf_header
     skip_before_action(:authenticate)
 
-    def index
+    def index # rubocop:disable Metrics/MethodLength
       render json: {
         meta: {
           versions: [
             {
-              version: '1.0.0',
+              version: '2.0.0',
               internal_only: false,
               status: VERSION_STATUS[:current],
+              path: '/services/claims/docs/v2/api',
+              healthcheck: '/services/claims/v2/healthcheck'
```

Comment:
doesnt this need to be wrapped in a feature flag so it doesnt show in sandbox/production?

Reply:
Yeah, I threw the docs stuff in this PR branch and ran out of time yesterday to get this figured out.
Will be adding it today.